src/command/FileCommands.cxx

```diff
@@ -234,9 +236,93 @@ handle_read_comments(Client &client, Request args, Response &r)
 	gcc_unreachable();
 }
 
+AllocatedPath
+find_art(AllocatedPath directory)
+{
+    std::vector<std::string> art_names = {
```

Comment:
Hi Max, thanks for the feedback. As I mentioned I have no real C++ experience, which presumably shows. What would be a more appropriate container for this string array?